src/Orleans/GrainDirectory/MultiClusterStatus.cs

```diff
@@ -6,7 +6,22 @@
     internal enum MultiClusterStatus : byte
```

Comment:
Because it only has a few values, and it is sent over the wire a lot, so there is no reason to use the  default representation which is int32